src/test/java/de/retest/recheck/review/workers/SaveFilterWorkerTest.java

```diff
@@ -21,8 +20,8 @@
 
 class SaveFilterWorkerTest {
 
-	@Rule
-	final TemporaryFolder folder = new TemporaryFolder();
+	@TempDir
+	Path base;
```

Comment:
This must go into setup:
```
@Before
void setup(@TempDir Path base) {
this.base=base;
}
```

Reply:
Why? Test passes and is IMHO more readable (less boilerplate).

Reply:
Oh, I just assumed that this was the cause why the build was red